README.md

```diff
@@ -72,8 +74,9 @@ This should create a `/home/pi/waveshare-epaper-display` directory.
 
     cd waveshare-epaper-display
     sudo apt update && sudo apt upgrade
-    sudo apt install git gsfonts python3 python3-pip cairosvg pigpio python3-pigpio
-    sudo pip3 install -r requirements.txt
+    sudo apt install gsfonts python3 python3-pip pigpio
+    python3 -m venv .venv
+    .venv/bin/pip3 install -r requirements.txt
```

Comment:
Virtual environment.  The reason is, the cryptography library has been a real pain.  Sometimes a cryptography version completely breaks pip and all pip install commands start to fail.  This way at least everything will be in a folder and is contained, and doesn't break the host operating system.